web/src/app/util/Options.js

```diff
@@ -45,24 +45,28 @@ export default class Options extends Component {
     errorMessage: '',
     showErrorDialog: false,
     showOptions: false,
+    ariaExpanded: false,
   }
 
   handleOpenMenu = event => {
     this.setState({
       anchorEl: event.currentTarget,
       show: true,
+      ariaExpanded: !this.state.ariaExpanded,
```

Comment:
Same as above, this is redundant, just check `anchorEl` wrap in `Boolean()` or prefix with `!!` if a bool is needed.